config/settings/development.yml

```
@@ -1,8 +1,6 @@
 dfe_signin:
-  issuer: https://signin-test-oidc-as.azurewebsites.net
-  profile: https://signin-test-pfl-as.azurewebsites.net
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
```

As a general pattern, it's best if we use random values for default values that are used for authentication, for security. Even if this isn't a deployed env, there's still risks involved I believe.

```suggestion
  secret: <%= SecureRandom.base64 %> # Override with SETTINGS__DFE_SIGNIN__SECRET
```

---

We don’t do that in any other settings file. Consistently wrong is better than inconsistent

---

But yes I agree

---

We are moving things over slowly. For example https://github.com/DFE-Digital/manage-courses-frontend/blob/2712-update-dsi-urls/config/settings.yml#L18

We can do this as a bigger move if you prefer.